lib/api/api-request.js

```diff
@@ -69,7 +73,7 @@ class RequestHandler extends AsyncResource {
     const { callback, opaque, abort, context } = this
 
     if (statusCode < 200) {
-        this.onInfo({statusCode,headers})
+      this.onInfo({ statusCode, headers })
```

Comment:
```suggestion
      if (this.onInfo) {
        this.onInfo({ statusCode, headers })
      }
```

Reply:
Wouldn't it be slower to add a boolean condition rather than relying on a noop function? That was the point of this design.

Reply:
conditions that are predicable are basically zero overhead